ZelBack/src/routes.js

```diff
@@ -1325,8 +1325,11 @@ module.exports = (app, expressWs) => {
   });
 
   // communication between multiple flux solution is on this:
-  app.ws('/ws/flux', (ws, req) => {
-    fluxCommunication.handleIncomingConnection(ws, req, expressWs.getWss('/ws/flux'));
+  app.ws('/ws/flux/:port', (ws, req) => {
+    fluxCommunication.handleIncomingConnection(ws, req, expressWs.getWss('/ws/flux/'));
+  });
+  app.ws('/ws/flux/', (ws, req) => {
```

Comment:
this line is not needed as its covered by above

Reply:
From my tests
app.ws('/ws/flux/:port', (ws, req) => {
    fluxCommunication.handleIncomingConnection(ws, req, expressWs.getWss('/ws/flux/'));
  });
is not triggered if port is not submited, resulted in no incoming connections on the node. Even with :port? I have got the same behaviour.